docs/registry/entities/openshift.md

```diff
@@ -0,0 +1,27 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/entity_namespace.md.j2 -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+# Openshift
+
+## Openshift Clusterquota
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+**type:** `openshift.clusterquota`
+
+**Description:** An OpenShift [ClusterResourceQuota](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/schedule_and_quota_apis/clusterresourcequota-quota-openshift-io-v1#clusterresourcequota-quota-openshift-io-v1) object.
+
+**Other Attributes:**
+
+> :warning: This entity definition contains attributes without a role.
```

Comment:
@joaopgrassi mentioned this, but please fix this.

Pick a `role` for each attribute on the entity:  `identifying` or `descriptive`

Reply:
done: [478b80e](https://github.com/open-telemetry/semantic-conventions/pull/2779/commits/478b80e4d45a51dcb70e2136c9e29322d5603631)